primitives/src/vrf.rs

```diff
@@ -166,6 +178,12 @@ pub mod sortition {
 		threshold.to_u128().expect("n * X, X ~ [0,1), n: u128; qed.")
 	}
 
+	pub fn is_selected(inout: &VRFInOut, threshold: u128) -> bool {
```

Comment:
Sort of off-topic, but do we guarantee that at least one prover will be selected for a given task? If not, what do we do if no one is selected?

Reply:
No, we need to choose metaparameters that make the probability of the sampled set {null} be negligible. For simplicity in the MVP, we let the task expire if that happens. E.g. with equal stakes, and just considering the sample size coefficient `c:=0.9`, the probability of having an empty set with a population size of 2 is 0.1^2. We have as requirement that tasks don't expire this way more than X% of the time. In this case, if X=1% then c:=0.9 for two participants is a valid solution.

Reply:
I see, good to know. It might be good to have this "limitation" (that tasks are not guaranteed execution) documented somewhere. It's also possible it's already documented and I just forgot lol